src/main/java/org/sonarlint/intellij/config/SonarLintProjectConfigurable.java

```diff
@@ -190,16 +189,18 @@ public void setOptions(Map<String, String> options) {
         for (Map.Entry<String, String> entry : options.entrySet()) {
           myRows.add(new KeyValuePair(entry.getKey(), entry.getValue()));
         }
-        Collections.sort(myRows, new Comparator<KeyValuePair>() {
-          @Override
-          public int compare(KeyValuePair o1, KeyValuePair o2) {
-            return o1.key.compareToIgnoreCase(o2.key);
-          }
-        });
+        Collections.sort(myRows, new KeyValueComparator());
         fireTableRowsInserted(0, options.size() - 1);
       }
     }
 
+    private static class KeyValueComparator implements Comparator<KeyValuePair> {
+        @Override
+        public int compare(KeyValuePair o1, KeyValuePair o2) {
+          return o1.key.compareToIgnoreCase(o2.key);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make this line start at column 9. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AIndentationCheck)